src/app/profile/overview/overview.component.html

```diff
@@ -1,40 +1,40 @@
 <div id="overview" class="container content">
-    <div class="row">
-        <div class="col-md-2">
-            <div class="row">
-                <img src="https://avatars2.githubusercontent.com/u/157761?v=3&s=460" width="150" height="250" />
-            </div>
-            <div class="row">
-                <h2>Pete Muir</h2>
-            </div>
-            <div class="row">
-                <p>
-                    pmuir
-                </p>
-                <p>
-                    I like writing clever one-line bios about myself. See? I just did.
-                </p>
-            </div>
-        </div>
-        <div class="col-md-10">
-            <div class="row">
-                <h1>Pinned Collaboration Spaces</h1>
-                <p>Customize</p>
-            </div>
-            <div *ngFor="let s of dummy.spaces; let i = index;" [ngClass]="{row: (i % 3 == 2) }">
-                <div class="col-md-4">
-                    <div class="panel panel-default">
-                        <div class="panel-body">
-                            <p>
-                                <a [routerLink]="[s.path]">{{s.name}}</a>
-                            </p>
-                            <p>
-                                {{s.description}}
-                            </p>
-                        </div>
-                    </div>
-                </div>
+  <div class="row">
+    <div class="col-md-2">
+      <div class="row">
+        <img src="{{context.current.entity.attributes.imageURL}}" width="150" height="250" />
+      </div>
+      <div class="row">
+        <h2>{{context.current.entity.attributes.fullName}}</h2>
+      </div>
+      <div class="row">
+        <p>
+          {{context.current.entity.attributes.username}}
+        </p>
+        <p>
+          {{context.current.entity.attributes.bio}}
```

Comment:
Yeah, I wasn't sure how to best do a cast here. I kinda need something like:

   {{ (context.current.entity as User).attributes.bio }}

Any ideas?